wpt/tests/browsing_context/command_get_tree.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC.
```

Comment:
For tests that we share with WPT we need to be careful about the license, so that we can share the tests under https://github.com/web-platform-tests/wpt/blob/master/LICENSE.md. As long as these tests aren't edited by anyone outside of Google we can just contribute them under the WPT license, but if there are contributions in this repo under the Apache license, then we couldn't just relicense those contributions without asking.

If we ultimately keep copies of wpt tests in the repo, then I'd suggest removing the copyright headers and adding the WPT license file in the wpt/ directory.

Reply:
thanks!